src/Symfony/Bundle/FrameworkBundle/Controller/RedirectController.php

```diff
@@ -89,11 +87,9 @@ public function redirectAction(Request $request, $route, $permanent = false, $ig
      * @param int|null    $httpPort  The HTTP port (null to keep the current one for the same scheme or the default configured port)
      * @param int|null    $httpsPort The HTTPS port (null to keep the current one for the same scheme or the default configured port)
      *
-     * @return Response A Response instance
```

Comment:
AFAIK, we are using an all-or-nothing strategy for the phpdoc of param and return until now: either we have nothing or everything

Reply:
I don't think we really enforce it, and I don't think we should enforce it: my feeling is that our policy is to add docblocks ony when the add value. Which means "partial" docblock should be really fine. It's perfectly valid for phpdocumentor btw.

Reply:
The less the comments feel like noise the better: people might even start reading them if you do that.